test_scripts/API/SetMediaClockTimer/ATF_P_SetMediaClockTimer_enableSeek_True.lua

```diff
@@ -0,0 +1,131 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [MOBILE API] SetMediaClockTimer request/response
+-- [HMI API] SetMediaClockTimer request/response
+--
+-- Description:
+-- MOBILE_API: <param name="enableSeek" type="Boolean" mandatory="false">
+-- HMI_API:  <param name="enableSeek" type="Boolean" mandatory="false">
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) SetMediaClockTimer RPC allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
```

Comment:
@OVikhrov, please check syntax in script, example:
c) App **successfylly**
 a) **Sens** SetMediaClockTimer